pkg/statistics/handle/ddl/subscriber.go

```diff
@@ -180,6 +245,19 @@ func (h handler) delayedDeleteStats4PhysicalID(
 	return errors.Trace(h.recordHistoricalStatsMeta(ctx, sctx, id, startTS))
 }
 
+func (h handler) insertStats4Col(
```

Comment:
```suggestion
func (h handler) insertStats2Col(
```
The inner function used 2.

Reply:
I will rename it in next PR

Reply:
the inner function is "insert column to KV", not "insert for column". So I think `4` is OK